### ISSUE TITLE
Why all the tutorial of model training will use trained model?

### ISSUE THREAD

**Issue by leekwunfung817** [matterport/Mask_RCNN#2723]
Why do all the tutorials of model training will use trained models? 
Theoretically, it will be affected by previous training data to cause it to be less accurate. 

1: How could I train a model without using a trained model?
2: Why not many people ask to train a pure new model for more accuracy?
3: Is that better?

**Comment by tbnxyzfz1** [comment]
You can comment out the load weight codes to train a new model. 
I tried this... after 80 epochs, the result is still not good. With a trained model, I can easily get a good model within 40 epochs.

**Comment by leekwunfung817** [comment]
> You can comment out the load weight codes to train a new model. I tried this... after 80 epochs, the result is still not good. With a trained model, I can easily get a good model within 40 epochs.

I want to inference new objects which totally do not exist in the trained model, did you train with the same set of objects?

**Comment by tbnxyzfz1** [comment]
> > You can comment out the load weight codes to train a new model. I tried this... after 80 epochs, the result is still not good. With a trained model, I can easily get a good model within 40 epochs.
> 
> I want to inference new objects which totally do not exist in the trained model, did you train with the same set of objects?

I used my own dataset. But I only have about 300 images.

**Comment by leekwunfung817** [comment]
> > > You can comment out the load weight codes to train a new model. I tried this... after 80 epochs, the result is still not good. With a trained model, I can easily get a good model within 40 epochs.
> > 
> > 
> > I want to inference new objects which totally do not exist in the trained model, did you train with the same set of objects?
> 
> I used my own dataset. But I only have about 300 images.

Do those data contain the previous existing classes in the trained model?

**Comment by nataliameira** [comment]
@leekwunfung817 

A good deep learning training uses thousands of images to learn its characteristics. Therefore, the pre-trained model has already learned many features, such as: borders, textures, colors, among others. So when you train your model in the last layers you won't need thousands of images. Hundreds of images will be enough.

**Comment by tbnxyzfz1** [comment]
> > > > You can comment out the load weight codes to train a new model. I tried this... after 80 epochs, the result is still not good. With a trained model, I can easily get a good model within 40 epochs.
> > > 
> > > 
> > > I want to inference new objects which totally do not exist in the trained model, did you train with the same set of objects?
> > 
> > 
> > I used my own dataset. But I only have about 300 images.
> 
> Do those data contain the previous existing classes in the trained model?

Not at all
